doc/modules/mod_mam.md

```diff
@@ -1,79 +1,83 @@
-# Module Description
-This module implements revision 0.2 of [XEP-0313 (Message Archive Management)](http://xmpp.org/extensions/attic/xep-0313-0.2.html). It is a highly customizable module, that requires some skill and knowledge to operate properly and efficiently.
+### Module Description
+This module implements revision 0.2 of [XEP-0313 (Message Archive Management)](http://xmpp.org/extensions/attic/xep-0313-0.2.html). It enables a service to store all users messages for one-to-one chats as well as group chats (MUC, MultiUser Chat). It uses [XEP-0059: Result Set Management](http://xmpp.org/extensions/xep-0059.html) for paging. It is a highly customizable module, that requires some skill and knowledge to operate properly and efficiently.
```

Comment:
This is no longer true. This module implements all current versions of MAM (from 0.2 to 0.5.1)